packages/@uppy/remote-sources/src/index.js

```diff
@@ -0,0 +1,78 @@
+import { BasePlugin } from '@uppy/core'
+import Dropbox from '@uppy/dropbox'
+import GoogleDrive from '@uppy/google-drive'
+import GooglePhotos from '@uppy/google-photos'
+import Instagram from '@uppy/instagram'
+import Facebook from '@uppy/facebook'
+import OneDrive from '@uppy/onedrive'
+import Box from '@uppy/box'
+import Unsplash from '@uppy/unsplash'
+import Url from '@uppy/url'
+import Zoom from '@uppy/zoom'
+
+import packageJson from '../package.json'
+
+const availablePlugins = {
+  // Using a null-prototype object to avoid prototype pollution.
+  __proto__: null,
+  Box,
+  Dropbox,
+  Facebook,
+  GoogleDrive,
+  GooglePhotos,
+  Instagram,
+  OneDrive,
+  Unsplash,
+  Url,
+  Zoom,
+}
+
+export default class RemoteSources extends BasePlugin {
```

Comment:
You're putting the `.js` file back, this should be `.ts`

Reply:
nice catch. must be a bad merge from main. I think this PR was originally created before the TS refactor happened.